src/pages/Payment/Payment.jsx

```diff
@@ -4,15 +4,17 @@ import ShippingAddress from './component/ShippingAddress';
 import './Payment.scss';
 
 const Payment = () => {
-  const [paymentPaymentProductList, setPaymentProductList] = useState([]);
+  const [PaymentProductList, setPaymentProductList] = useState([]);
```

Comment:
```suggestion
  const [paymentProductList, setPaymentProductList] = useState([]);
```
state는 camelCase로 해주세요~
그리고 
paymentProductList로
상품금액,배송비, 총결제금액, 포인트차감 이부분에 쓰셔야할것같아요
state는 선언되어있는데 사용하고있는 부분은 없어서요.

Reply:
PR 이미지 마크업 수정하고 피드백 반영했습니다!